### ISSUE TITLE
Using Zend search breaks site

### ISSUE THREAD

**Issue by dannyuk1982** [statamic/v2-hub#1037]
### Expected behaviour
Zend search should work and not break the entire site

### Actual behaviour
The site breaks

### Steps to reproduce
1.  `php please search:update` - local/storage/search gets set to 0755 and the files within 0666

2.  go to the CP and save a page, this error appears: `There was a problem saving the data. Please check your logs.`

3.  go to any page on the site and it's broken, with this error
```
Whoops, looks like something went wrong.

1/1
RuntimeException in Filesystem.php line 55:
```

### Server configuration
**Operating system**:
Centos 7.2

**Web server:**
Apache

**PHP version:**
PHP 7.0.11

**Statamic version:**
2.1.15

**Comment by jackmcdade** [comment]
So you're saying the search update actually affects your permissions?

**Comment by dannyuk1982** [comment]
Yea, if I set them to 0777 running search update changes them to 0755 / 0666

**Comment by edalzell** [comment]
it must delete and recreate that directory and your PHP user is diff than your apache user

**Comment by dannyuk1982** [comment]
`ps aux | egrep '(apache|httpd)'` and `php -r 'echo get_current_user();'` both return the same user

**Comment by edalzell** [comment]
Hmmm...755 & 666 should be fine if they're the same user, I think.

**Comment by dannyuk1982** [comment]
it gives the error in the OP, this on,y seems to be triggered by saving a CP item (only page tested) though

**Comment by dannyuk1982** [comment]
Might be fixed, will test in the wild and resubmit if still there

**Comment by jasonvarga** [comment]
The bug fix in 2.1.15 was targeted to your issues specifically. The problem may still exist, however now the site shouldn't break. You will probably see the exception in your logs.

**Comment by dannyuk1982** [comment]
Cheers Jason
